build/Targets/Packages.props

```diff
@@ -55,10 +55,10 @@
     <MicrosoftMSXMLVersion>8.0.0.0-alpha</MicrosoftMSXMLVersion>
     <!-- Using a private build of Microsoft.Net.Test.SDK to work around issue https://github.com/Microsoft/vstest/issues/373 -->
     <MicrosoftNETTestSdkVersion>15.6.0-dev</MicrosoftNETTestSdkVersion>
-    <MicrosoftNetCompilersVersion>2.6.0</MicrosoftNetCompilersVersion>
-    <MicrosoftNetRoslynDiagnosticsVersion>2.6.0-beta2</MicrosoftNetRoslynDiagnosticsVersion>
+    <MicrosoftNetCompilersVersion>2.6.1</MicrosoftNetCompilersVersion>
+    <MicrosoftNetRoslynDiagnosticsVersion>2.6.1-beta1-62702-01</MicrosoftNetRoslynDiagnosticsVersion>
     <MicrosoftNetCoreILAsmVersion>2.0.0</MicrosoftNetCoreILAsmVersion>
-    <MicrosoftNETCoreCompilersVersion>2.8.0-beta2-62712-07</MicrosoftNETCoreCompilersVersion>
+    <MicrosoftNETCoreCompilersVersion>2.7.0-beta3-62720-08</MicrosoftNETCoreCompilersVersion>
```

Comment:
Why are we downgrading these packages?

Reply:
@jmarolf I have tried lot of different versions of this package, including 2.8.X version, on a separate PR (https://github.com/dotnet/roslyn/pull/25179), but none of them seem to work. I tried the latest 2.7 version as Jason recommended trying it out, but that [did not work](https://ci.dot.net/job/dotnet_roslyn/job/master/job/windows_coreclr_release_prtest/441/consoleFull#119230013679494335-f7bd-47d0-8771-8661e00c2db2) either. I have reverted the downgrade to see if it works now. At this point, I am happy to put any version of this package that unblocks this PR.

Reply:
I would like to understand the dependencies here. @jaredpar what is `MicrosoftNETCoreCompilersVersion` and why is it different from `MicrosoftNetCompilersVersion`?